script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt

```diff
@@ -16,17 +16,17 @@ You see, I was wondering if you're in some kind of trouble that's causing you to
 "It doesn't matter if it's boring.
 I just want to hear it."
 "...Hah. You're very curious, aren't you?"
-"I guess so", she laughs.
+'I guess so', she laughs.
 <Page6>
 "Then let's continue our conversation.
-You said you were returning to your own home, but you've never gone back to your parents' house?"
+You said you were returning to your own home, but you've never gone back to your family home?"
 <Page7>
-"Yes. For various reasons, I haven't been back to my parents' house for almost seven years. It's like a life of recuperation...or something like that.
+"Yes. For various reasons, I haven't been back to my family home for almost seven years. It's like a life of recuperation...or something like that.
```

Comment:
Maybe just change this to "haven't been back **there** for almost", so that we don't dupe? It's a new page at least I guess